CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+### Submitting Patches
+
+We welcome PRs to this project.  Please have a look at `git log` and try
+to match the commit log style.  (Note: we aren't strict about style, but
+it would be nice if the commit messages were well constructed and
+informative.)
+
```

Comment:
This was a handy reference: https://chris.beams.io/posts/git-commit/#seven-rules